lib/main.js

```diff
@@ -32,23 +46,31 @@ export async function main(
   // Get the installation ID
   // https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#get-a-repository-installation-for-the-authenticated-app
   const { data: installation } = await request(
-    "GET /repos/{owner}/{repo}/installation",
+    "GET /orgs/{org}/installation",
```

Comment:
if we need to get the installation ID for the current owner, we should keep doing it using `GET /repos/{owner}/{repo}/installation`. Because if the owner is set to something else, we need to check two endpoints, because the owner can be either an org or a user:

1. [GET /orgs/{org}/installation](https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#get-a-repository-installation-for-the-authenticated-app)
2. [GET /users/{username}/installation](https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#get-a-user-installation-for-the-authenticated-app)

Reply:
See my suggested change at https://github.com/timreimherr/create-github-app-token/pull/1